packages/express-openapi/package.json

```diff
@@ -33,7 +33,6 @@
   },
   "devDependencies": {
     "@types/ajv": "^1.0.0",
-    "@types/express": "^4.16.0",
-    "multer": "^1.4.1"
```

Comment:
i don't see any harm in having tests using multer.  it isn't going to effect users in production, and it does provide some checking to ensure that the framework _can work_ with file attachments.